torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1976,6 +1977,29 @@ Error PyTorchModelLoader::loadMM(const torch::jit::Node *ptNode) {
   return addValueMapping(outputs[0], output);
 }
 
+Error PyTorchModelLoader::loadBmm(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 2, outputs, 1));
+
+  glow::NodeValue lhs;
+  ASSIGN_VALUE_OR_RETURN_ERR(lhs, getGlowNodeValueForValue(inputs[0]));
+  glow::NodeValue rhs;
+  ASSIGN_VALUE_OR_RETURN_ERR(rhs, getGlowNodeValueForValue(inputs[1]));
+
+  // Check dimensions of inputs
+  if (lhs.dims().size() != 3 || rhs.dims().size() != 3) {
```

Comment:
I think RHS could be either 2 or 3 dimensional. createBatchMatMul works for both. Otherwise looks good to me!

Reply:
I'm going off the documentation here https://pytorch.org/docs/stable/torch.html#torch.bmm, does that seem right?

Reply:
aten::bmm works for both. Looks like the pt op imposes an additional constraint. Since we are dealing with only pt models we should be good.